Gemfile

```diff
@@ -39,6 +39,7 @@ gem "inifile",                        "~>3.0",         :require => false
 gem "kubeclient",                     "~>2.4.0",       :require => false # For scaling pods at runtime
 gem "manageiq-api-client",            "~>0.1.0",       :require => false
 gem "manageiq-network_discovery",     "~>0.1.2",       :require => false
+gem "manageiq-messaging", :git => "https://github.com/ManageIQ/manageiq-messaging", :branch => "master", :require => false
```

Comment:
> This is not a plugin...it's a released gem, do should not use manageiq_plugin

its not a _released_ gem, https://rubygems.org/gems/manageiq-messaging => 404 😢 

maybe we can release it?!

Reply:
Gah, yeah, it's *supposed* to be released :)